### DISCUSSION TITLE
Context aware announcement bar

### DISCUSSION THREAD

**Discussion by saihaj** [facebook/docusaurus#3892] [Q&A] [answered]
I am working on a project (https://github.com/shabados/docs/issues/10) where we are migrating docs from all our repos to single repo and use Docusaurus v2. Since we have multiple projects in our docs site is it possible to have context aware [announcement bars](https://v2.docusaurus.io/docs/api/themes/configuration/#announcement-bar)? 

So suppose I have my site structured as
```
/ <- Homepage for all docs
/project-1  <- docs for project repo 1
/project-2 <- docs for project repo 2
```

So when I am at homepage I want to show some announcement. When I am on `/project-1` then I want to show announcements regrading next major release and on `/project-2` I want to show star our repo banner.

**Comment by slorber** [comment, marked as answer]
Hi,

I don't think we support that, but you can probably swizzle the component and add your own project logic here
